cmake/aditof-setup.iss.cmakein

```diff
@@ -15,7 +15,7 @@ AppSupportURL="http://www.analog.com"
 AppUpdatesURL="http://www.analog.com"
 AppCopyright="Copyright 2019 Analog Devices, Inc."
 CreateAppDir=yes
-LicenseFile="C:\projects\aditof-sdk\LICENSE"
+LicenseFile="D:\a\1\s\LICENSE"
```

Comment:
You can get the path to the repo by using cmake. This way to get rid of harcoded paths. Without harcoded paths, one can easily generate it's own installed on his/her pc.

Reply:
Fixed.